SPECS/python-idna/python-idna.spec

```diff
@@ -17,10 +17,14 @@ Support for the Internationalised Domain Names in Applications (IDNA) protocol a
 %package -n     python3-idna
 Summary:        Internationalized Domain Names in Applications (IDNA).
 BuildRequires:  python3-devel
+BuildRequires:  python3-pip
 BuildRequires:  python3-setuptools
 BuildRequires:  python3-xml
+BuildRequires:  python-flit-core
```

Comment:
Small ask: could you put the BRs back in alphabetical order to keep it neat?

Reply:
On it!